Contrib/mmpa/test_rfrag.py

```diff
@@ -0,0 +1,25 @@
+#
+#  Copyright (C) 2017 greg Landrum
+#
+#   @@ All Rights Reserved  @@
+#
+import unittest, subprocess, os
+from rdkit import RDConfig
+
+
+class TestCase(unittest.TestCase):
+  def test1Github1406(self):
+    with open('data/simple.smi') as inf:
+      p = subprocess.run(('python', 'rfrag.py'),stdin=inf,stdout=subprocess.PIPE)
```

Comment:
In general, I think it's not  good idea to start a subprocess in a unit test for performance reasons. Here, it's ok.